packages/grafana-ui/src/components/MatchersUI/FieldsByFrameRefIdMatcher.test.tsx

```diff
@@ -109,4 +109,10 @@ describe('RefIDMultiPicker', () => {
     expect(mockOnChange).toHaveBeenLastCalledWith(['A', 'B']);
     /* eslint-enable testing-library/prefer-user-event */
   });
+
+  it('Should display a refID that does not exist in the selection', async () => {
```

Comment:
Nit: 
- We're not using `should` in our other test names in this file.
- We could make this more descriptive, to explain why we're showing refIDs that don't exist. Otherwise this sounds a bit of a mental test!
```suggestion
  // This RefID does not exist now but did previously, when it was selected and saved
  it('displays a refID that does not exist in the selection', async () => {
```